web/src/components/ChatSecurityInformation.tsx

```diff
@@ -21,8 +23,8 @@ const SecurityIconContainer = styled.button`
 `
 
 const SecurityIcon = styled(Icon)`
-  width: 32px;
-  height: 32px;
+  width: 100%;
+  height: 100%;
```

Comment:
🤔 Why did you change this?

Reply:
You can download and slow down the video.

https://github.com/user-attachments/assets/4ee4ec1f-f826-4a33-8938-24b344b3e88b